src/utils/useProfileRedirect.tsx

```diff
@@ -0,0 +1,18 @@
+import { useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import useAuth from './useAuth';
+
+function useProfileRedirect() {
+  const navigate = useNavigate();
+  const { isAuthenticated } = useAuth();
+
+  useEffect(() => {
+    console.log('useProfileRedirect.fired');
```

Comment:
Before creating a release branch to merge into `main`, be sure to strip out console.log statements!